llama_stack/providers/adapters/evals/eleuther/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
should move this over to impls - we use adapters only for remote. I think we should just rename adapters to "remote" move impls to "inline".